includes/Integrations/Core_Themes_Support.php

```diff
@@ -79,11 +80,12 @@ public function extend_theme_support() {
 	 * @return void
 	 */
 	public function embed_web_stories() {
-		$customizer = new Customizer();
-		$this->enqueue_style( 'web-stories-theme-style-' . get_stylesheet(), [ Renderer::STYLE_HANDLE ] );
+		if ( is_readable( WEBSTORIES_PLUGIN_DIR_PATH . '/assets/css/web-stories-theme-style-' . get_stylesheet() . '.css' ) ) {
```

Comment:
Why was this added?

Reply:
We don't have stylesheets for some themes like `2019`, there it was going 404. so better to check if the file exists.